backend-rust/src/migrations/m0013-baker-metrics.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE metrics_bakers
```

Comment:
Some documentation would be nice.
Also the fact that this table only contain rows for blocks where these have changed and not for every block.